src/components/ExampleForm.vue

```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable -->
```

Comment:
Eslint disabled

Reply:
Will be resolved the same way as we are going to agree on the case above.